app/src/views/CookiePolicy/index.tsx

```diff
@@ -0,0 +1,380 @@
+                <div>
+                    <Container
+                        heading={strings.disclaimerTitle}
+                        footerIcons={strings.disclaimerDescription}
+                        withHeaderBorder
+                        withInternalPadding
+                        containerRef={declaimerRef}
+                    />
+                    <Container
+                        heading={strings.useOfOurInformationTitle}
+                        containerRef={useOfOurInformationRef}
+                        footerContentClassName={styles.headerDescription}
+                        footerContent={(
+                            <>
+                                <div>{strings.useOfOurInformationDescription1}</div>
+                                <div>
+                                    {!accepted && resolveToComponent(
+                                        strings.useOfOurInformationDescription2,
+                                        {
+                                            termsLink: (
+                                                <Link
+                                                    href="/"
+                                                    withUnderline
+                                                    external
+                                                >
+                                                    {strings.useOfOurInformationAudiovisualLink}
+                                                </Link>
+                                            ),
+                                        },
+                                    )}
+                                </div>
+                                <div>{strings.useOfOurInformationDescription3}</div>
+                                <div>
+                                    <Link
+                                        href="https://www.ifrc.org/fraudulent-emails-and-websites"
+                                        withUnderline
+                                        external
+                                    >
+                                        {strings.useOfOurInformationDescriptionLink}
+                                    </Link>
+                                </div>
+
+                            </>
+                        )}
+                        withHeaderBorder
+                        withInternalPadding
+                    />
+                    <Container
+                        heading={strings.ourPrivacyPolicyHeading}
+                        withHeaderBorder
+                        withInternalPadding
+                        containerRef={ourPrivacyPolicyRef}
+                    >
+                        <div>
+                            {resolveToString(strings.ourPrivacyPolicyContent, {
+                                publishedDay: 'November',
+                                publishedDate: 29,
+                                publishedYear: 2021,
+                            })}
```

Comment:
Same as above

Reply:
@barshathakuri While leaving of comments and reviews, let's not use "Same as above" as the context might get lost because of multiple reasons.

Reply:
Noted!